### PR TITLE
chore: update docker build, ci

### PR DESCRIPTION
**Pull request by Vid201**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,40 +1,55 @@
...
 RUN apk add openssl1.1-compat
 
-COPY --from=builder /silius/target/release/silius /usr/local/bin/silius
+# Copy silus binary
+COPY --from=builder /app/silius /usr/local/bin/silius
 
+# Copy licenses
+COPY LICENSE-* ./
+
+# Expose ports
```
Comment (zsluedem) [inline Dockerfile:52]: We could build "locally" and just copy the target into the image. In this case, docker image is much smaller.

File: Dockerfile [hunk 2]
```diff
@@ -1,40 +1,55 @@
-# build
-FROM ubuntu:18.04 AS builder
+FROM lukemathwalker/cargo-chef:latest-rust-1 AS chef
```
Comment (zsluedem) [inline Dockerfile:1]: We could build "locally" and just copy the target into the image. In this case, docker image is much smaller.

Comment (Vid201) [reply Dockerfile:1]: That's true. But I would keep the building inside Docker so someone who can't build Silius locally (due to unsupported architecture) can build it in Docker.

**Review by zsluedem — APPROVED** [review]
LGTM

**Pull request merged**